### ISSUE TITLE
Zoom / radio frequency buttons for the G3000 are swapped. 

### ISSUE THREAD

**Issue by jackwaller57** [Working-Title-MSFS-Mods/fspackages#1135]
I am having the following issue with G3000 in the longitude. When I go to use the physical buttons, the two stacked buttons that are meant to change the radio frequency seem to affect the zoom instead. When I go to the middle button that should affect the zoom, rather than zoom it lets me change the value of the radio frequency for the digits to the right of the decimal place. When I go to use the far left physical button it changes the radio frequency for the digits to the left of the decimal place.

**Comment by cywang90** [comment]
Yes, this is a known issue. The problem is that in the real longitude, the left "knob" is actually a joystick that can be pressed down and tilted in all four directions (like the PFD joystick), thus allowing it to be used to control map zoom/scroll functions. In the sim it's not modeled this way and can only be turned CW and CCW. So to preserve map control functionality I have left the knob assignments the way Asobo had them set up originally as a compromise.
